test/rubygems/test_require.rb

```diff
@@ -296,6 +300,22 @@ def test_default_gem_only
     assert_equal %w(default-2.0.0.0), loaded_spec_names
   end
 
+  def test_default_gem_require_activates_just_once
+    default_gem_spec = new_default_spec("default", "2.0.0.0",
+                                        nil, "default/gem.rb")
+    install_default_specs(default_gem_spec)
+
+    assert_require "default/gem"
+
+    times_called = 0
+
+    Kernel.stub(:gem, ->(name, requirement) { times_called += 1 }) do
+      refute_require "default/gem"
+    end
+
+    assert_equal 0, times_called
```

Comment:
im not entirely understanding what we expect to be called 0 times?

Reply:
I expect `Kernel.gem` to no longer be called after the first required. Currently it's called every time you require a default gem.